### PR TITLE
LPS-178683 Control Panel: Revert the code of the front-end part

### PR DESCRIPTION
**Pull request by Stephy6**
Motivation
=======
It has been decided to stop segmentation unification, so for that, as a tango team we have decided to revert the code from master and move it to the branch with name LPS-154036-unified-segmentation-experience. The reason to do that, is to prevent problems in the future. The code involved in this PR is related to the epic and is the front-end part [LPS-166954-Migrate the Segmentation Editor to the Control Panel](https://issues.liferay.com/browse/LPS-166954) 

[Link to story or ticket](https://issues.liferay.com/browse/LPS-178496)
[Link to technical task](https://issues.liferay.com/browse/LPS-178074)
[Link to technical subtask](https://issues.liferay.com/browse/LPS-178683)

Proposed Solution
========
1. Revert all code from master below FF related to the epic: https://issues.liferay.com/browse/LPS-166954

Steps to Verify:
----------------
1. Keep the FF LPS-166954
2. Go to Control Panel -> Segments
3. Create various new Segments of different contributors (User Segment, Organization Segment, Session Segment..) and of different types (Dates, String, Boolean..) and complete their necessary fields.
4. Check that the Scope Card isn't present anymore.
5. Save the Segment/s, re-open it and check that the values are correct and it doesn't throw any error.

### CODE REVIEW

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorsBuilder.es.js [hunk 1]
```diff
@@ -45,11 +42,9 @@ class ContributorBuilder extends React.Component {
 		onConjunctionChange: PropTypes.func,
 		onPreviewMembers: PropTypes.func,
 		onQueryChange: PropTypes.func,
-		portletNamespace: PropTypes.string.isRequired,
 		previewMembersURL: PropTypes.string,
 		propertyGroups: PropTypes.arrayOf(propertyGroupShape),
 		renderEmptyValuesErrors: PropTypes.bool,
-		scopeName: PropTypes.string,
```
Comment (claraizquierdo) [inline modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorsBuilder.es.js:52]: I believe that you can also remove the siteItemSelectorURL.

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorsBuilder.es.js [hunk 2]
```diff
@@ -184,341 +150,133 @@ class ContributorBuilder extends React.Component {
...
-												displayType="secondary"
-												showCollapseIcon
-											>
-												<ClayPanel.Body className="p-4">
-													{this.props
-														.siteItemSelectorURL && (
-														<div className="align-items-center d-flex justify-content-between mb-3">
-															<p className="mb-0 mr-6 text-dark">
-																{Liferay.Language.get(
-																	'select-the-scope-of-your-segment-to-specify-where-it-can-be-used'
```
Comment (claraizquierdo) [inline modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorsBuilder.es.js:211]: Could this translation be removed from the language.properties file?

**Review by claraizquierdo — CHANGES_REQUESTED** [review]
Hi @Stephy6,

I've detected two little things. I have made the review by having a look at the changes we made in this PR: https://github.com/liferay-tango/liferay-portal/pull/3087/files#diff-73189f4cd752a5d39c96c208b8c1dae848c38b958264ce375b3b9e295454731f  If you could have a look at it in case I missed something it would be great 😄

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorsBuilder.es.js (hunk 2 above)
Comment (Stephy6) [reply modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorsBuilder.es.js:211]: True, thank you!

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorsBuilder.es.js (hunk 1 above)
Comment (Stephy6) [reply modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorsBuilder.es.js:52]: ahh you're right, thank you!

**Comment by Stephy6** [comment]
ci.test:sf

**Comment by Stephy6** [comment]
ci:test:relevant

**Comment by Stephy6** [comment]
ci:test:sf

**Comment by Stephy6** [comment]
ci:test:sf

**Review by claraizquierdo — APPROVED** [review]
LGTM 🎉

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Comment by claraizquierdo** [comment]
ci:test:tango

**Comment by ealonso** [comment]
ci:close

**Comment by ealonso** [comment]
see: https://github.com/liferay-tango/liferay-portal/pull/3342

**Pull request closed without merging**